adagrams/game.py

```diff
@@ -1,13 +1,115 @@
 from random import randint
 
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    #make a new list named pool_list
+    pool_list = []
+    for key, value in LETTER_POOL.items():
+        for i in range(0,value):
+            pool_list.append(key)
+    
+
+    #draw ten times from pool_list randomly and remove the tile from
+    #the pool_list
+    drawn_list = []
+    total_index = len(pool_list)
+    draw_time = 10
+    for i in range(0,draw_time):
+        index = randint(0,total_index-1)
+        element = pool_list.pop(index)
+        drawn_list.append(element)
+        total_index -= 1
```

Comment:
Rather than holding a separate variable that we need to manually keep in sync with the length of the letter pool, we should directly use `len(pool_list)` any time we need the length of that list. This reduces the possibility of bugs that could occur if `total_index` ever go out of sync with `pool_list`.

```suggestion
    draw_time = 10
    for i in range(0,draw_time):
        index = randint(0, len(pool_list) - 1)
        element = pool_list.pop(index)
        drawn_list.append(element)
```